openshift/apigateway.yml

```diff
@@ -0,0 +1,137 @@
+########################################
+## API Gateway ConfigMap
+########################################
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: apigateway
+  namespace: openwhisk
```

Comment:
Namespace should be left off here like the other OpenShift configs.

Reply:
done!